infra/binder/src/main/java/org/apache/shardingsphere/infra/binder/engine/statement/dml/InsertStatementBinder.java

```diff
@@ -63,6 +64,12 @@ private InsertStatement copy(final InsertStatement sqlStatement) {
         result.getValues().addAll(sqlStatement.getValues());
         sqlStatement.getSetAssignment().ifPresent(result::setSetAssignment);
         sqlStatement.getOnDuplicateKeyColumns().ifPresent(result::setOnDuplicateKeyColumns);
+        sqlStatement.getOnConflictKeyColumns().ifPresent(segment -> {
```

Comment:
Please do not use PostgreSQLInsertStatement in SQLBinder. If you need call setOnConflictKeyColumnsSegment method, please add it to InsertStatement, and implement setOnConflictKeyColumnsSegment in PostgreSQLInsertStatement.